src/main/kotlin/pt/up/fe/ni/website/backend/model/PostDTO.kt

```diff
@@ -0,0 +1,9 @@
+package pt.up.fe.ni.website.backend.model
+
+import java.util.Date
+
+class PostDTO {
+    val description: String? = null
+    val link: String? = null
+    val date: Date? = null
+}
```

Comment:
In order to avoid bloating the model with DTO's, I think this class should just be in the `PostService.kt` file since it's intrinsic to the service. I know we usually follow a "one class per file" approach in Java but I don't think we need to follow it here in Kotlin and it seems more organized to me